src/components/PromoSheet/index.tsx

```diff
@@ -1,64 +1,72 @@
-import lang from 'i18n-js';
 import React, { useCallback, useEffect, useReducer } from 'react';
 import { StatusBar } from 'react-native';
-import {
-  SheetActionButton,
-  SheetHandle,
-  SlackSheet,
-} from '../../components/sheet';
-import InfoRow from './InfoRow';
+import LinearGradient from 'react-native-linear-gradient';
+import MaskedView from '@react-native-masked-view/masked-view';
+import { SheetActionButton, SheetHandle, SlackSheet } from '@/components/sheet';
 import { CampaignKey } from '@/campaigns/campaignChecks';
 import { ImgixImage } from '@/components/images';
 import { analytics } from '@/analytics';
 import {
   AccentColorProvider,
   Box,
-  Heading,
   Inset,
   Row,
   Rows,
   Stack,
   Text,
+  Bleed,
+  Column,
+  Columns,
 } from '@/design-system';
 import { useDimensions } from '@/hooks';
-import { useNavigation } from '@/navigation';
 import { sharedCoolModalTopOffset } from '@/navigation/config';
 import { useTheme } from '@/theme';
 
 const MIN_HEIGHT = 740;
 
-interface PromoSheetProps {
+type SheetActionButtonProps = {
```

Comment:
also why type vs interface? i've just been doing interface for props structuring by default

Reply:
Oh I just default to `type`. I can switch this back to be consistent, but I think 9 times out of 10 type is preferable since you can't extend them (I think this is a feature, unless it's actually needed), and you can use `type`s to describe more than just objects i.e. tuples, functions, etc.